android/build.gradle

```diff
@@ -34,6 +34,6 @@ android {
 }
 
 dependencies {
-    implementation 'com.instabug.library:instabug:8.4.0.0'
+    implementation 'com.instabug.library:instabug:8.4.0.1'
```

Comment:
Since we will release after merging, we should update the latest version 8.4.0.6.